### ISSUE TITLE
IB/implicit examples -- problem with input files

### ISSUE THREAD

**Issue by deleeke** [IBAMR/IBAMR#146]
All the IB/implicit examples compile fine, but the examples fail with the message that  they lack any definition of `stokes_ib_precond_db` required by  `src/IB/IBImplicitStaggeredHierarchyIntegrator.cpp`
 
The reference I find to that is in line 175 in `IBImplicitStaggeredHierarchyIntegrator`:     
``` 
   if (!d_solve_for_position)
    {
        Pointer<Database> stokes_ib_pc_db = input_db->getDatabase("stokes_ib_precond_db");
        d_stokes_solver = new IBImplicitStaggeredStokesSolver(object_name, stokes_ib_pc_db);
        ins_hier_integrator->setStokesSolver(d_stokes_solver);
    }
```
What are valid entries for `stokes_ib_precond_db` in the input file?
What would be a sane default in `IBImplicitStaggeredHierarchyIntegrator.cpp` for this if no preconditioner is provided in the input database? 
*(The problem is this value is being referenced without any check to see if it exists!)*

**Comment by boyceg** [comment]
I am going to remove those implicit examples for now. We do have implicit tests (which are more experimental) elsewhere in the repo.

In principle, we should be able to use the same examples for both explicit and implicit solvers.

**Comment by boyceg** [comment]
"Fixed" by #169
